biggus/tests/unit/test_Array.py

```diff
@@ -96,7 +96,7 @@ def _test(self, shape, dtype, expected):
 
     def test_0d(self):
         self._test((), 'f8',
-                   "<Array shape=() dtype=dtype('float64') size=8 B>")
+                   "<Array shape=(,) dtype=dtype('float64') size=8 B>")
```

Comment:
Hmmm, this is problematic - ``(,)`` doesn't represent anything you can create in Python.